docs/source/trainer/using_the_trainer.rst

```diff
@@ -271,6 +271,25 @@ engineering. We currently support the ``cpu`` and ``gpu`` devices.
         device='cpu'
     )
 
+TRaining on TPU
+~~~~~~~~~~~~~~~
+
+Now you can also use composer to train your model on single core ``tpus``
+in ``bf16`` precision. You will need to have ``torch_xla`` installed using
+instructions here https://github.com/pytorch/xla.
+
+.. testcode::
```

Comment:
this shouldn't be testcode since we don't actually run on TPU during our doctets